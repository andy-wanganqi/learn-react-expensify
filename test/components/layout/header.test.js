import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../../src/components/layout/header.jsx';

test('Should render Header', () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
