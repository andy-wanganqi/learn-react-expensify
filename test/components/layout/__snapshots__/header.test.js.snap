// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <div>
    <NavLink
      className=""
      to="/"
    >
      Go home
    </NavLink>
    <NavLink
      className=""
      to="/create"
    >
      Add Expense
    </NavLink>
    <NavLink
      className=""
      to="/edit"
    >
      Edit Expense
    </NavLink>
    <NavLink
      className=""
      to="/help"
    >
      Help
    </NavLink>
  </div>
</header>
`;
